spec/gateways/logging_api_spec.rb

```diff
@@ -0,0 +1,65 @@
+describe Gateways::LoggingApi do
+  describe '#search' do
+    subject { described_class.new }
+
+    context "with correct data" do
+      let(:username) { "AAAAA" }
+      let(:logs) do
+        [
+          {
+            "ap" => "",
+            "building_identifier" => "",
+            "id" => 1,
+            "mac" => "",
+            "siteIP" => "",
+            "start" => "2018-10-01 18:18:09 +0000",
+            "stop" => nil,
+            "success" => true,
+            "username" => username
+          },
+          {
+            "ap" => "",
+            "building_identifier" => "",
+            "id" => 1,
+            "mac" => "",
+            "siteIP" => "",
+            "start" => "2018-10-01 18:18:09 +0000",
+            "stop" => nil,
+            "success" => true,
+            "username" => username
+          }
+        ]
+      end
+
+      before do
+        stub_request(:get, "http://govwifi-logging-api.com/authentication/events/search?username=#{username}").
```

Comment:
This seems good too - happy to use webmock when testing a "remote service"-style gateway.